src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj

```diff
@@ -27,6 +32,18 @@
   <ItemGroup>
     <Compile Include="System\Diagnostics\StackFrameExtensions.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">  
+    <Compile Include="System\Diagnostics\StackTraceSymbols.CoreCLR.cs" />  
+    <ProjectReference Include="..\..\System.Collections\src\System.Collections.csproj" />  
+    <ProjectReference Include="..\..\System.Threading.Overlapped\src\System.Threading.Overlapped.csproj" />  
```

Comment:
Eric, just so I understand as well, the feedback is to guard the P2P reference to System.Threading.Overlapped to happen only when $(TargetGroup) == '' and $(OSGroup) == 'Windows_NT'?